lib/v2/bangumi/tv/other/followrank.js

```diff
@@ -0,0 +1,48 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const url = 'https://bgm.tv/anime';
+    const response = await got({
+        url,
+        method: 'get',
+        headers: {
+            'User-Agent': 'RSSHub (https://docs.rsshub.app/routes/anime#bangumi-fan-zu-ji-hua)',
+        },
```

Comment:
The default user agent(a randomised chrome on mac) works. Is there any specific reasons to change it?

Reply:
yeah. the offical wanna know which program access their website. [link](https://github.com/bangumi/api/blob/master/docs-raw/user%20agent.md)

Reply:
https://github.com/DIYgod/RSSHub/blob/05d53e01487ca8d0594e5d9c9f64e61ab8176473/lib/config.js#L5

Reply:
done. Thanks for your patience.